bundles/org.eclipse.ui.navigator/schema/navigatorContent.exsd

```diff
@@ -385,6 +385,13 @@ If the required action provider is disabled or unbound to the current viewer, th
                </appInfo>
             </annotation>
          </attribute>
+         <attribute name="muiltiprojectEnable" type="string">
+            <annotation>
+               <documentation>
+                  Used to specify how action handles multi-project selection and action enablement.  The value &quot;any&quot; means if any project is enabled, then action provider is enabled.
```

Comment:
The sentences aren't really grammatically complete.  Perhaps more like this:

Used to specify how the action handles multi-project selection for action enablement.  The value &quot;any&quot; means if any project is enabled, then the action provider is enabled.

Reply:
Ok, will change

Reply:
Done